### PR TITLE
samples: Bluetooth: Add NORDIC SDK APP START and END tags

### PR DESCRIPTION
**Pull request by joerchan**
Add NORDIC SDK APP START and NORDIC SDK APP END tags to Bluetooth
samples that did not have them.

Signed-off-by: Joakim Andersson <joakim.andersson@nordicsemi.no>

### CODE REVIEW

**Review by trond-snekvik — APPROVED** [review]
Just out of curiosity, what is the purpose of these tags?

**Comment by joerchan** [comment]
> Just out of curiosity, what is the purpose of these tags?

It's used by SES, that's all I know.

**Review by lemrey — APPROVED** [review]

File: samples/bluetooth/shell_bt_nus/CMakeLists.txt [hunk 1]
```diff
@@ -8,8 +8,10 @@ cmake_minimum_required(VERSION 3.13.1)
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 project(NONE)
 
+# NORDIC SDK APP START
 target_sources(app PRIVATE
   src/main.c
 )
 
 zephyr_library_include_directories(.)
+# NORDIC SDK APP END
```
Comment (tejlmand) [inline samples/bluetooth/shell_bt_nus/CMakeLists.txt:17]: This should be moved up.
SES-NE doesn't recognize `zephyr_library_include_directories()`

Btw, why is there a `zephyr_library_include_directories()` in the sample CMakeLists.txt ?

**Review by tejlmand — CHANGES_REQUESTED** [review]

File: samples/bluetooth/shell_bt_nus/CMakeLists.txt (hunk 1 above)
Comment (joerchan) [reply samples/bluetooth/shell_bt_nus/CMakeLists.txt:17]: @tejlmand I don't know. Sounds like you know better, should it be removed? Or replaced with something else?

Comment (tejlmand) [reply samples/bluetooth/shell_bt_nus/CMakeLists.txt:17]: the `app` lib is a Zephyr library, and is defined as last library in the boilerplate, so it is possible to do:
```
zephyr_library_sources(...)
zephyr_library_include_directories(...) 
```
or:
```
target_sources(...)
target_include_directories(...) 
```

both are actually valid, but in general `zephyr_library` should be used with care, as a new zephyr library created AFTER boilerplate behaves differently.

Also, it looks strange to mix the two ways in the same file.
Guess the code is copied from somewhere else.

If SES-NE's feature of defining include directories is used, then `target_include_directories` will be added to this file, in which case this file will contain both `zephyr_library_include_directories` and `target_include_directories(app ...)`

So my suggestion is to switch to `target_include_directories(app PRIVATE .)` and keep the code inside the tags.

Comment (joerchan) [reply samples/bluetooth/shell_bt_nus/CMakeLists.txt:17]: Thanks. Updated to your suggestion.

**Review by tejlmand — APPROVED** [review]

**Pull request merged**